test/integration/index.spec.js

```diff
@@ -184,7 +201,38 @@ describe('/functions', () => {
 
     assert.deepStrictEqual(functionById, functionFiltered)
   })
+  it('POST', async () => {
+    const { data: newFunc } = await axios.post(`${URL}/functions`, func)
+    assert.equal(newFunc.name, 'test_func')
```

Comment:
@i-pip `assert.equal` is deprecated so I recommend updating:

```suggestion
    assert.strictEqual(newFunc.name, 'test_func')
```

Reply:
I just substituted all occurrences of `assert.equal` with `assert.strictEqual` - I hope that's okay